CommonLibSF/include/RE/T/TESObjectCELL.h

```diff
@@ -12,7 +13,7 @@ namespace RE
 	public:
 		SF_RTTI_VTABLE(TESObjectCELL);
 		SF_FORMTYPE(CELL);
-
-		//
+		bool           pad[0xD8];
+		TESWorldSpace* CellWorldspace;
```

Comment:
please use camelCase for member variables (ie. `cellWorldspace`) and add a comment with its offset.

Reply:
done, thanks for the comments.